app/models/colyseus-models/pokemon.ts

```diff
@@ -13772,6 +13793,27 @@ export class WormadamSandy extends Pokemon {
       !regionSynergies.includes(Synergy.GRASS)
     )
   }
+
+  onEvolve({
+    pokemonEvolved: mothim,
+    pokemonsBeforeEvolution: wormadam
+  }: {
+    pokemonEvolved: Pokemon
+    pokemonsBeforeEvolution: Pokemon[]
+  }) {
+    const preEvolve = wormadam.pop()
```

Comment:
prefer non-mutative method `.at(-1)` ; `wormadams[0]` should work as well i guess